sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py

```diff
@@ -89,6 +91,8 @@ def parse_starting_position(args):
 recv_cnt_iteration_map = defaultdict(int)
 recv_time_map = dict()
 
+azure_metric_monitor = AzureMonitorMetric("ASync EventHubConsumerClient", "ASync EventHubConsumerClient")
```

Comment:
ASync?

Reply:
yep, this is the async consumer test file, so the name being "async"

Reply:
updated!